private-templates-service/client/src/utils/AllCardsUtil/Scroller.ts

```diff
@@ -0,0 +1,36 @@
+export enum ScrollDirection {
+  Vertical,
+  Horizontal
+}
+export class Scroller {
+  timeStamp: number;
+
+  _scroll: (item: EventTarget & Element, deltaScroll: number, mult: number) => void;
+  constructor(scrollDirection: ScrollDirection = ScrollDirection.Vertical) {
+    this.timeStamp = 0;
+    if (scrollDirection === ScrollDirection.Vertical) {
+      this._scroll = this.verticalScroll;
+    } else {
+      this._scroll = this.horizontalScroll;
+    }
+  }
+  horizontalScroll = (item: EventTarget & Element, deltaScroll: number, mult: number) => {
+    if (deltaScroll > 0) item.scrollLeft += 10 * mult;
+    else item.scrollLeft -= 10 * mult;
+  };
+
+  verticalScroll = (item: EventTarget & Element, deltaScroll: number, mult: number) => {
+    if (deltaScroll > 0) item.scrollTop += 10 * mult;
+    else item.scrollTop -= 10 * mult;
```

Comment:
It would be great if these constants were put into a `const` at the top of this file.